src/cmd/cli/main.go

```diff
@@ -949,6 +951,11 @@ func main() {
 	}
 
 	flushAllTracking()
+
+	if len(os.Args) > 2 && os.Args[1] == "gen-docs" && os.Args[2] != "" {
```

Comment:
If we convert this to the usual Cobra way of doing things, I'm OK to merge it. Ie. `var docsCmd = …` (can be hidden from `defang help`)